datasource/opennms-kafka/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,12 +12,36 @@
             <cm:property name="edgesTopic" value="edges"/>
             <cm:property name="inventoryTtlMs" value="86400000"/> <!-- 24 hours -->
             <cm:property name="inventoryGcIntervalMs" value="300000"/> <!-- 5 minutes -->
+            <cm:property name="scriptFile" value=""/>
```

Comment:
Should include a comment here about the behavior of the empty value.

Reply:
done